src/Symfony/Component/Routing/Requirement/Requirement.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Requirement;
+
+/**
+ * A collection of universal regular-expression constants to use as route parameter requirements.
+ */
+enum Requirement
```

Comment:
Why is this an `enum`?

Reply:
Having a fake nein seems odd to me 
, why not add a private empty constructor instead?

Reply:
Adding final + private constructor is a lot of boilerplate that using `enum` makes useless. It's odd because it's new :)

Reply:
Okay, maybe I just have to get used to that.